diff -urpN a/arch/arm/tools/mach-types b/arch/arm/tools/mach-types
--- a/arch/arm/tools/mach-types	2023-05-19 14:36:40.135127756 +0800
+++ b/arch/arm/tools/mach-types	2023-05-19 14:47:10.378740564 +0800
@@ -1006,6 +1006,7 @@ eco5_bx2		MACH_ECO5_BX2		ECO5_BX2		4572
 eukrea_cpuimx28sd	MACH_EUKREA_CPUIMX28SD	EUKREA_CPUIMX28SD	4573
 domotab			MACH_DOMOTAB		DOMOTAB			4574
 pfla03			MACH_PFLA03		PFLA03			4575
+nuc980			MACH_NUC980		NUC980			4577
 #
 # Additional mach-types supported by OpenWrt
 #
